utils/ssh.py

```diff
@@ -480,6 +480,14 @@ def set_initial_file_end(self):
             fstat = sshtail._sftp_client.stat(self._remote_filename)
             self._remote_file_size = fstat.st_size  # Seed initial size of file
 
+    def raw_output(self):
+        """Returns the output of the log as one (raw) string"""
+        return '\n'.join(self.lines_as_list())
```

Comment:
1. No it is not exactly the same.

        (line for line in something)

 is a generator

      [line for line in something]

 is a list